.github/workflows/run-archiver.yml

```diff
@@ -50,38 +47,43 @@ jobs:
           ZENODO_SANDBOX_TOKEN_UPLOAD: ${{ secrets.ZENODO_SANDBOX_TOKEN_UPLOAD }}
           ZENODO_SANDBOX_TOKEN_PUBLISH: ${{ secrets.ZENODO_SANDBOX_TOKEN_PUBLISH }}
           EPACEMS_API_KEY: ${{ secrets.EPACEMS_API_KEY }}
-          # They're in the repository secrets, but we need more QA/QC checks
-          # ZENODO_TOKEN_UPLOAD: ${{ secrets.ZENODO_TOKEN_UPLOAD }}
-          # ZENODO_TOKEN_PUBLISH: ${{ secrets.ZENODO_TOKEN_PUBLISH }}
+          ZENODO_TOKEN_UPLOAD: ${{ secrets.ZENODO_TOKEN_UPLOAD }}
+          ZENODO_TOKEN_PUBLISH: ${{ secrets.ZENODO_TOKEN_PUBLISH }}
         run: |
-          pudl_archiver --sandbox --datasets ${{ matrix.dataset }} --summary-file ${{ matrix.dataset }}_run_summary.json
+          pudl_archiver --sandbox --datasets ${{ matrix.dataset }} --dry-run --summary-file ${{ matrix.dataset }}_run_summary.json
 
-      - name: Archive code coverage results
+      - name: Upload run summaries
+        id: upload_summaries
         uses: actions/upload-artifact@v3
         with:
-          name: $${{ matrix.dataset }}-run-summary
+          name: run-summaries
           path: ${{ matrix.dataset }}_run_summary.json
 
   archive-notify:
     runs-on: ubuntu-latest
     needs: archive-run
     if: ${{ always() }}
     steps:
+      - uses: actions/checkout@v4
+      - name: Download summaries
+        id: download
+        uses: actions/download-artifact@v3
+        with:
+          name: run-summaries
+      - name: show summaries
+        run: ls -R
+      - name: Munge summaries together
+        id: all_summaries
+        run: |
+          {
+            echo "SLACK_PAYLOAD<<EOF"
+            ./scripts/make_slack_notification_message.py --summary-files *_run_summary.json | tee slack-payload.json
```

Comment:
I started with using `jq` to munge a bunch of stuff since I didn't want to check out the repo, but that became an unreadable mess quickly.